src/DynamoCore/Properties/Resources.Designer.cs

```diff
@@ -1301,7 +1301,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to New sequence.
+        ///   Looks up a localized string similar to type: var[]..[] (New list).
```

Comment:
The format we're following is that the type is in brackets like @Amoursol notes below.

Reply:
Addressed in f347775